invenio_deposit/api.py

```diff
@@ -76,7 +80,11 @@ def wrapper(self_or_cls, *args, **kwargs):
 
 
 def has_status(method=None, status='draft'):
-    """Check that deposit has defined status (default: draft)."""
+    """Check that deposit has a defined status (default: draft).
+
+    :param method: Function executed if record has a defined status.
+    :params status: Defined status to check.
```

Comment:
I'd use `param` instead of `params`.

http://www.sphinx-doc.org/en/stable/domains.html#info-field-lists